demo/SFSW2_project_descriptions.R

```diff
@@ -54,10 +54,10 @@ project_paths <- list(
 
   # Path to where large outputs are saved to disk
   dir_big = dir_big <- if (identical(opt_platform[["host"]], "local")) {
-      dir_prj
-    } else if (identical(opt_platform[["host"]], "hpc")) {
-      dir_prj
-    },
+    dir_prj
```

Comment:
I am not sure which tab is correct here. Can you just double check that yours is correct?